src/NServiceBus.Core/Persistence/Raven/TimeoutPersister/RavenTimeoutPersistence.cs

```diff
@@ -10,7 +10,7 @@ namespace NServiceBus.Persistence.Raven.TimeoutPersister
     using Logging;
     using Timeout.Core;
 
-    public class RavenTimeoutPersistence : IPersistTimeouts
+    public class RavenTimeoutPersistence : IPersistTimeouts, IPersistTimeoutsV2
```

Comment:
looks so, although the NSB.RavenDB package supports back to Core 4.1. Should we change that?